docs/local_usage.md

```diff
@@ -0,0 +1,48 @@
+# Running the bridge locally and testing it end to end manually from UI yourself
+
+## Local extension Setup:
+ - The tests setup a ganache server with a preset mnemonic. Add the first key from this mnemonic to your Metamask as an extra account to use for testing. You can find this first key here: ARTEMIS_ETHEREUM_KEY in /polkadot-ethereum/test/docker-compose.yml
```

Comment:
I think it will add more context if we add links to [ganachee](https://www.trufflesuite.com/ganache) and [Metask](https://docs.metamask.io/guide/#why-metamask) for users who are not familiar with them.